ui/components/app/transaction-breakdown/transaction-breakdown.component.js

```diff
@@ -23,6 +23,8 @@ export default class TransactionBreakdown extends PureComponent {
     gasPrice: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
     gasUsed: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
     totalInHex: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    baseFee: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
I left some comments [here](https://www.figma.com/file/NXvUKcZ0Ocu1GPcT7VxRMx/1559---Gas-Controls?node-id=1410%3A22096), but I think we are missing some fields that will be important to the user and there's some question about what this refers to (baseFee of the block versus maxFee)

Reply:
per the figma / slack discussion, lets add 'effectiveGasPrice' that will also be a number.